webui/react/src/components/InfoBox.module.scss

```diff
@@ -35,6 +35,10 @@
       flex-grow: 1;
       font-size: 14px;
       margin: 0;
+
+      .blank {
+        background-color: var(--theme-stage-strong);
```

Comment:
this doesn't really show up in dark mode

Reply:
:+1: